jetbrains-core/src/software/aws/toolkits/jetbrains/utils/CoroutineUtils.kt

```diff
@@ -7,14 +7,27 @@ import com.intellij.openapi.Disposable
 import com.intellij.openapi.application.AppUIExecutor
 import com.intellij.openapi.application.ModalityState
 import com.intellij.openapi.application.impl.coroutineDispatchingContext
+import com.intellij.openapi.util.Disposer
+import kotlinx.coroutines.coroutineScope
+
+// FIX_WHEN_MIN_IS_2019_3 this can be removed and the actual runUnlessDisposed can be used which has more features
```

Comment:
since this is a clone of the actual should this go in CompatabilityUtils instead?